numcosmo/Makefile.am

```diff
@@ -453,7 +453,8 @@ nc_headers = \
         lss/nc_galaxy_redshift_gauss.h         \
         lss/nc_galaxy_wl.h                     \
         lss/nc_galaxy_wl_dist.h                \
-        lss/nc_galaxy_wl_reduced_shear_gauss.h \
+        lss/nc_galaxy_wl_ellipticity_gauss.h \
```

Comment:
Please use the correct indentation, no tabs and align the \ (remember to not put anything after \).